src/main/java/frc/robot/commands/AimAtTarget.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Navigation;
+import frc.robot.subsystems.drive.Drive;
+
+public class AimAtTarget extends Command {
+    Drive driveSubsystem;
+    Navigation navSubsystem;
+    int targetId;
+
+    public AimAtTarget(Drive driveSubsystem, Navigation navSubsystem, int targetId) {
```

Comment:
This command should have a `end()` method that puts the drivetrain back into the MANUAL state when the command ends